test/msw-api/handle-stream.test.ts

```diff
@@ -0,0 +1,60 @@
+import * as path from 'path'
+import { pageWith } from 'page-with'
+import { waitUntil } from '../support/utils'
+
+function createRuntime() {
+  return pageWith({
+    example: path.resolve(__dirname, 'handle-stream.mocks.ts'),
+    routes(app) {
+      app.get('/stream', (req, res) => {
+        res.write('first-chunk')
+
+        setTimeout(() => {
+          res.write(' last-chunk')
+          res.end()
+        }, 1000 * 6)
```

Comment:
6s is a rather long timeout. Can we make it lower while still preserving the async nature of the stream in the test?

Perhaps `250 * 6`?

Reply:
yes of course